Windows/SCPCB.sln

```diff
@@ -191,22 +280,126 @@ Global
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.MinSizeRel|x86.Build.0 = Debug|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.Release|x64.ActiveCfg = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.Release|x64.Build.0 = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.Release|x86.ActiveCfg = Release|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.Release|x86.Build.0 = Release|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.ReleaseDLL|x64.ActiveCfg = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.ReleaseDLL|x64.Build.0 = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.ReleaseDLL|x86.ActiveCfg = Release|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.ReleaseDLL|x86.Build.0 = Release|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.RelWithDebInfo|x64.ActiveCfg = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.RelWithDebInfo|x64.Build.0 = Release|x64
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.RelWithDebInfo|x86.ActiveCfg = Release|Win32
+		{5833EEA1-1068-431F-A6E5-316E7DC5D90A}.RelWithDebInfo|x86.Build.0 = Release|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Debug|x64.ActiveCfg = Debug|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Debug|x64.Build.0 = Debug|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Debug|x86.ActiveCfg = Debug|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Debug|x86.Build.0 = Debug|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.DebugDLL|x64.ActiveCfg = DebugDLL|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.DebugDLL|x64.Build.0 = DebugDLL|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.DebugDLL|x86.ActiveCfg = DebugDLL|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.DebugDLL|x86.Build.0 = DebugDLL|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.MinSizeRel|x64.ActiveCfg = Debug|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.MinSizeRel|x64.Build.0 = Debug|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.MinSizeRel|x86.ActiveCfg = Debug|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.MinSizeRel|x86.Build.0 = Debug|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Release|x64.ActiveCfg = Release|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Release|x64.Build.0 = Release|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Release|x86.ActiveCfg = Release|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.Release|x86.Build.0 = Release|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.ReleaseDLL|x64.ActiveCfg = ReleaseDLL|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.ReleaseDLL|x64.Build.0 = ReleaseDLL|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.ReleaseDLL|x86.ActiveCfg = ReleaseDLL|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.ReleaseDLL|x86.Build.0 = ReleaseDLL|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.RelWithDebInfo|x64.ActiveCfg = Release|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.RelWithDebInfo|x64.Build.0 = Release|x64
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.RelWithDebInfo|x86.ActiveCfg = Release|Win32
+		{AFF27A26-C088-444B-BC2A-0BA94A02AFA7}.RelWithDebInfo|x86.Build.0 = Release|Win32
```

Comment:
We do not want any of these additional configurations.

Reply:
These appear to have come from libogg.